src/libsync/discovery.cpp

```diff
@@ -1219,6 +1219,9 @@ DiscoverySingleDirectoryJob *ProcessDirectoryJob::startAsyncServerQuery()
 
 bool ProcessDirectoryJob::runLocalQuery()
 {
+    QByteArray localPath = (_discoveryData->_localDir + _currentFolder._local).toUtf8();
```

Comment:
@ogoffart Where's this new path used?

Reply:
Happens. 

I think it would look somewhat nicer if all the logging and error strings further down also used `localPath` over the longer string conat. But that's optional.